src/main/java/in/projecteka/gateway/consent/ConsentArtefactController.java

```diff
@@ -0,0 +1,39 @@
+package in.projecteka.gateway.consent;
+
+import in.projecteka.gateway.clients.HipConsentNotifyServiceClient;
+import in.projecteka.gateway.clients.HiuConsentNotifyServiceClient;
+import in.projecteka.gateway.common.RequestOrchestrator;
+import lombok.AllArgsConstructor;
+import org.springframework.http.HttpEntity;
+import org.springframework.http.HttpStatus;
+import org.springframework.web.bind.annotation.PostMapping;
+import org.springframework.web.bind.annotation.ResponseStatus;
+import org.springframework.web.bind.annotation.RestController;
+import reactor.core.publisher.Mono;
+
+import static in.projecteka.gateway.common.Constants.X_HIP_ID;
+import static in.projecteka.gateway.common.Constants.X_HIU_ID;
+
+
+@RestController
+@AllArgsConstructor
+public class ConsentArtefactController {
+    RequestOrchestrator<HipConsentNotifyServiceClient> hipConsentNotifyRequestOrchestrator;
+    RequestOrchestrator<HiuConsentNotifyServiceClient> hiuConsentNotifyRequestOrchestrator;
+
+    @ResponseStatus(HttpStatus.ACCEPTED)
+    @PostMapping("/v1/consents/hip/notify")
+    public Mono<Void> consentNotifyToHIP(HttpEntity<String> requestEntity) {
+        Mono<Void> toBeFiredAndForgotten = hipConsentNotifyRequestOrchestrator.processRequest(requestEntity, X_HIP_ID);
```

Comment:
you can do the subscribe inside the processRequest itself as it's kind of pattern we are doing everwhere.

Reply:
Mounica changed something as part of her story in some ConsentRequestController, refer that please just in case.

Reply:
Sure, but all the tests needs to be changed as part of this. So thinking of raising a separate PR if thats okay.

Reply:
Sure :)